int8 message		# data about connection settings
---
bool ack 			# acknowledged informations